app/javascript/packs/initializers/initializeDateTimeHelpers.js

```diff
@@ -1,8 +1,6 @@
-/* global localizeTimeElements */
+import { localizeTimeElements } from '../../utilities/localDateTime';
```

Comment:
Separately from this PR, might be a good idea to review if the `localDateTime.js` that's under `app/assets/javascripts` is still required once the initializer migrations are all done